mail_debrand/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,7 @@
+* Rafael Blasco <rafabn@tecnativa.com>
```

Comment:
As the module was totally redone, contributors were reset, and this one (and the following) hasn't contributed in current one. Any way, the email address is incorrect.

Reply:
Fixed!